src/converters/fromZigbee.ts

```diff
@@ -6323,6 +6323,26 @@ const converters2 = {
             }
         },
     } as Fz.Converter,
+    nodon_fil_pilote_mode: {
+        cluster: 'manuSpecificNodOnFilPilote',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload: KeyValueAny = {};
+            const mode = msg.data['0'];
```

Comment:
Tt shouldn't be `const mode = msg.data['mode'];` ?

Reply:
Yes you're right ! I seen it this morning I didn't had time to update my PR to fix it

Reply:
Done here [https://github.com/Koenkk/zigbee-herdsman-converters/pull/6465](https://github.com/Koenkk/zigbee-herdsman-converters/pull/6465)